db/schema.rb

```diff
@@ -54,7 +54,7 @@
     t.index ["username", "domain"], name: "index_accounts_on_username_and_domain", unique: true, using: :btree
   end
 
-  create_table "block_mutes", force: :cascade, id: false do |t|
+  create_table "block_mutes", force: :cascade do |t|
```

Comment:
If so, migration may also be necessary. Because it was not there, I left it as it was.